src/store/flow/block-types/MobilePrimitives_SelectOneResponseBlockStore.ts

```diff
@@ -51,31 +51,37 @@ export const getters: GetterTree<IFlowsState, IRootState> = {
 }
 
 export const mutations: MutationTree<IFlowsState> = {
-  deleteChoiceByKey(state, { choiceKeyToRemove, blockId }) {
```

Comment:
As we don't have access to rootState  from Mutation & this.state.flow throws ts exception, I turned this into Actions